File: webservice/classes/point.php
<?php

Class point{
    
    private $_id;
    private $_longitude;
    private $_latitude;
    private $_laZone;
    
    function get_id() {
        return $this->_id;
    }

    function get_longitude() {
        return $this->_longitude;
    }

    function get_latitude() {
        return $this->_latitude;
    }

    function get_laZone() {
        return $this->_laZone;
    }

    function set_id($_id) {
        $this->_id = $_id;
    }

    function set_longitude($_longitude) {
        $this->_longitude = $_longitude;
    }

    function set_latitude($_latitude) {
        $this->_latitude = $_latitude;
    }

    function set_laZone($_laZone) {
        $this->_laZone = $_laZone;
    }

    function __construct($_id = null, $_longitude = null, $_latitude = null, $_laZone = null) {
        $this->_id = $_id;
        $this->_longitude = $_longitude;
        $this->_latitude = $_latitude;
        $this->_laZone = $_laZone;
    }

    
    
}

File: android-admin/BetaMap/MyApplication2/app/src/main/java/com/gaming/beehaku/myapplication/Classes/Point.java
package com.gaming.beehaku.myapplication.Classes;

/**
 * Created by Beehaku on 19/05/2016.
 */
public class Point {

    private int _id;
    private Float _latitude;
    private Float _longitude;
    private int theArea;

    public Point(int _id, Float _latitude, Float _longitude, int theArea) {
        this._id = _id;
        this._latitude = _latitude;
        this._longitude = _longitude;
        this.theArea = theArea;
    }

    public Point() {

    }

    public int GetId() {
        return _id;
    }

    public void SetId(int _id) {
        this._id = _id;
    }

    public Float GetLatitude() {
        return _latitude;
    }

    public void SetLatitude(Float _latitude) {
        this._latitude = _latitude;
    }

    public Float GetLongitude() {
        return _longitude;
    }

    public void SetLongitude(Float _longitude) {
        this._longitude = _longitude;
    }

    public int GetTheArea() {
        return theArea;
    }

    public void SetTheArea(int theArea) {
        this.theArea = theArea;
    }
}
